modcc/module.cpp

```diff
@@ -209,119 +209,21 @@ bool Module::semantic() {
         return true;
     };
 
+    // Add built in function that approximate exp use pade polynomials
+    functions_.push_back(
+        Parser{"FUNCTION exp_pade_11(z) { exp_pade_11=(1+0.5*z)/(1-0.5*z) }"
+        }.parse_function());
+    functions_.push_back(
+        Parser{ "FUNCTION exp_pade_22(z)"
+        "{ exp_pade_22=(1+0.5*z+0.08333333333333333*z*z)/(1-0.5*z+0.08333333333333333*z*z) }"
+        }.parse_function());
+
```

Comment:
yeah, I can fix it. It isn't immediately clear to me how to indent code inside strings...